src/components/Icon/Icon.css

```diff
@@ -0,0 +1,18 @@
+/* Using BEM naming scheme: http://getbem.com/introduction/ */
```

Comment:
Note that I'm using BEM here. Suggest doing so throughout the app. Also suggest considering moving to [CSS modules](https://facebook.github.io/create-react-app/docs/adding-a-css-modules-stylesheet#docsNav), which are supported by create react app.

Reply:
Discussing BEM tomorrow.